Jenkinsfile.rh

```diff
@@ -14,46 +14,51 @@
  * limitations under the License.
  */
 @Library(['private-pipeline-library', 'jenkins-shared']) _
+import com.sonatype.jenkins.shared.Expectation
 
-properties([
-  parameters([
-    string(
-      name: 'version',
-      description: 'Version tag to apply to the image, like 1.140.0-ubi-1.'
-    ),
-  ]),
-])
-
-node('ubuntu-zion') {
-  try {
-    stage('Preparation') {
-      deleteDir()
-
-      checkout scm
-
-      sh 'docker system prune -a -f'
-      sh '''
-        wget -q -O preflight \
-          https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.4.1/preflight-linux-amd64
-        chmod 755 preflight
-      '''
-    }
-    stage('Build') {
-      withCredentials([
-        usernamePassword(
-            credentialsId: 'red-hat-quay-nexus-iq-server',
-            usernameVariable: 'REGISTRY_LOGIN',
-            passwordVariable: 'REGISTRY_PASSWORD'),
-        string(
-            credentialsId: 'red-hat-api-token',
-            variable: 'API_TOKEN')
-      ]) {
-        sh 'PATH="$PATH:." VERSION=$version ./build_red_hat_image.sh'
-      }
-    }
-  } finally {
-    sh 'docker logout'
-    sh 'docker system prune -a -f'
-    sh 'git clean -f && git reset --hard origin/master'
+dockerizedBuildPipeline(
+  prepare: {
+    githubStatusUpdate('pending')
+  },
+  setVersion: {
+    env['VERSION'] = "${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
+  },
```

Comment:
Can drop for the snapshot build

Reply:
Addressed: https://github.com/sonatype/docker-nexus-iq-server/pull/123/commits/78a503dbcd229917ff3cb39b0464d42f9a68d9e2